tasks/main.yml

```diff
@@ -20,16 +20,32 @@
   become: True
   with_items: "{{ beegfs_kmod_preload }}"
 
-- name: Stop all beegfs services
+- name: Generate /etc/hosts file
+  template:
+    src: hosts.j2
+    dest: /etc/hosts
+  become: yes
```

Comment:
This is something that should be listed as a prerequisite and handled outside of the role.

Reply:
I added a comment in my review but it's not here - I think this is out of scope for BeeGFS and should be handled eg by cluster_setup in p3-appliances.